fizz-cli/fizz_cli/main.py

```diff
@@ -21,6 +21,7 @@
 from .utils import save_yaml_file
```

Comment:
Add BlackConnect plugin to your ide then reformat and push this file again. If you don't want to do that then don't format the entire file.